azuretest/inputsources/shared/blueprint.ftl

```diff
@@ -67,6 +67,17 @@
                         "Name" : "userpool"
                     }
                 }
+            },
+            "Tiers" : {
+                "mgmt" : {
+                    "Components" : {
+                        "baseline" : {
+                            "baseline" : {
+                                "azure:AdministratorGroups" : [ "1234567890-1234567890-1234567890-1234567890" ]
+                            }
+                        }
+                    }
```

Comment:
Should this be in the mock input rather than the shared one? This would be the default value whenever someone doesn't set the value

Reply:
I thought azuretest was all mock. Have I misunderstood what azuretest is?

Reply:
Ah right missed that sorry, yeah its ok to go here 
The other place that might be better is in the baseline testing module, all the other components will still use it and keeps the baseline configuration in a centralised point

Reply:
Will do